### PR TITLE
Reduce complexity of adding new relic

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Fixing the build by reducing the complexity of the create_app method

### How to review 
Does the build pass?

### CODE REVIEW

**Review by ajmaddaford — APPROVED** [review]

**Review by benjefferies — APPROVED** [review]
I think create_app can be tidied up a lot more and we should look at doing it properly at some point but I understand this is a blocking issue for now

File: README.md [hunk 1]
```diff
@@ -250,6 +250,7 @@ EQ_MAX_HTTP_POST_CONTENT_LENGTH - The maximum http post content length that the
 EQ_MAX_NUM_REPEATS - The maximum number of repeats the system will allow
 EQ_DEVELOPER_LOGGING - Enable developer style logging described here http://structlog.readthedocs.io/en/stable/development.html
 
+EQ_NEW_RELIC_ENABLED - Enable New Relic monitoring
 NEW_RELIC_LICENSE_KEY - Enable new relic monitoring by supplying a New Relic licence key
 NEW_RELIC_APP_NAME - The name to display for the application in New Relic
```
Comment (benjefferies) [inline README.md:255]: Is this something that needs to be configurable?

File: tests/integration/test_application_variables.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+from mock import patch
```
Comment (benjefferies) [inline tests/integration/test_application_variables.py]: Unused import?

**Review by benjefferies — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jonnyshaw89) [reply README.md:255]: What do you mean?

Comment (benjefferies) [reply README.md:255]: We could remove NEW_RELIC_APP_NAME  and do

```
def setup_newrelic():
    if settings.EQ_NEW_RELIC_ENABLED:
        import newrelic.agent
        newrelic.agent.initialize()
        newrelic.agent.register_application(name='eq-survey-runner')
```

Comment (jonnyshaw89) [reply README.md:255]: We could but what if we wanted to log both pre-prod and prod to the same New Relic account?

Comment (benjefferies) [reply README.md:255]: I guess that's what NEW_RELIC_ENVIRONMENT is for according to https://docs.newrelic.com/docs/agents/python-agent/customization-extension/python-management-api#initialize

Comment (jonnyshaw89) [reply README.md:255]: They are much the same but this just means we don't need to add the ini file

Comment (benjefferies) [reply README.md:255]: Fair enough if they are. I didn't know if new relic had a view where you could see all of the same apps in their different environments or something.

**Review by benjefferies — APPROVED** [review]

**Pull request merged**